resources/config.test.edn

```diff
@@ -36,4 +36,11 @@
                                              :upgrade-from         "1.1"
                                              :channels             {:channel-1 {:worker-count [10 :int]
                                                                                 :retry        {:count   [5 :int]
-                                                                                               :enabled [true :bool]}}}}}}}
\ No newline at end of file
+                                                                                               :enabled [true :bool]}}}}}
+            :producer             {:default {:bootstrap-servers       "localhost:9092"
```

Comment:
What is this `default` key?

Reply:
@kartik7153 `default` just refers to the default KafkaProducer config. Let me know if this needs change.